src/services/endpoints/pattern.service.js

```diff
@@ -0,0 +1,134 @@
+import { SDKName } from '../../config';
+import util from '../../util';
+import debug from 'debug';
+
+export default class SpectrumEventsService {
+	constructor(client) {
+		this._debug    = debug(`${SDKName}:spectrum:events`);
```

Comment:
`spectrum:events` should be `endpoints:pattern` or anything else that's related to this class.